src/app/Main.java

```diff
@@ -59,5 +61,17 @@ public static void main(String[] args){
         application.setLocationRelativeTo(null); // app opens on center of screen
         application.pack();
         application.setVisible(true);
+
+        Game game = new CommonGame("pop", "hard", 10, 3);
+        Song song = new CommonSong("Closer", "The Chainsmokers", new OnlineMP3PlayableAudio("path/song.mp3"));
+        Round round = new TextInputRound(song, "What song is this?", "Closer");
+        game.setCurrentRound(round);
+        gameDataAccessObject.save(game);
+
+        RoundState roundState = roundViewModel.getState();
+        roundState.setGenre("pop");
+        roundState.setMaxRounds(10);
+        roundState.setInitialLives(3);
+        roundState.setGameId(game.getID());
```

Comment:
I believe this can be cleared to avoid any redundant and unnecessary merge conflicts.

Reply:
Thank you I forgot to remove it again after testing the last changes